### PR TITLE
python310Packages.snowflake-connector-python: 2.8.0 -> 2.8.2

### PR DESCRIPTION
**Pull request by dotlambda**
###### Description of changes
fixes CVE-2022-42965
https://github.com/snowflakedb/snowflake-connector-python/blob/v2.8.2/DESCRIPTION.md

###### Things done

- Built on platform(s)
  - [x] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- [ ] For non-Linux: Is `sandbox = true` set in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [ ] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [22.11 Release Notes (or backporting 22.05 Release notes)](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#generating-2211-release-notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
  - [ ] (Release notes changes) Ran `nixos/doc/manual/md-to-db.sh` to update generated release notes
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

File: pkgs/development/python-modules/snowflake-sqlalchemy/default.nix [hunk 1]
```diff
@@ -9,14 +9,14 @@
 
 buildPythonPackage rec {
   pname = "snowflake-sqlalchemy";
-  version = "1.4.3";
+  version = "1.4.4";
```
Comment (LeSuisse) [inline pkgs/development/python-modules/snowflake-sqlalchemy/default.nix:12]: Is this upgrade needed for the upgrade of `snowflake-connector-python`? The release notes of the two libs did not make that obvious.

**Review by LeSuisse — COMMENTED** [review]
Changelogs/release notes seem fine and `nixpkgs-review` is happy.


Result of `nixpkgs-review pr 203398` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>4 packages built:</summary>
  <ul>
    <li>python310Packages.snowflake-connector-python</li>
    <li>python310Packages.snowflake-sqlalchemy</li>
    <li>python39Packages.snowflake-connector-python</li>
    <li>python39Packages.snowflake-sqlalchemy</li>
  </ul>
</details>

File: pkgs/development/python-modules/snowflake-sqlalchemy/default.nix (hunk 1 above)
Comment (dotlambda) [reply pkgs/development/python-modules/snowflake-sqlalchemy/default.nix:12]: No but it's a bugfix that can be backported anyway.

Comment (LeSuisse) [reply pkgs/development/python-modules/snowflake-sqlalchemy/default.nix:12]: Thanks for the confirmation, I was not sure if I missed something or not

**Review by LeSuisse — APPROVED** [review]

**Review by risicle — APPROVED** [review]
`nixpkgs-review` happy, macos 10.15, nixos x86_64 & aarch64-linux

**Pull request merged**